docs/devel/scheduler_algorithm.md

```diff
@@ -47,6 +47,7 @@ The purpose of filtering the nodes is to filter out the nodes that do not meet c
 - `PodFitsHost`: Filter out all nodes except the one specified in the PodSpec's NodeName field.
 - `PodSelectorMatches`: Check if the labels of the node match the labels specified in the Pod's `nodeSelector` field ([Here](../user-guide/node-selection/) is an example of how to use `nodeSelector` field).
 - `CheckNodeLabelPresence`: Check if all the specified labels exist on a node or not, regardless of the value.
+- `MaxEBSVolumeCount`: Ensure that the number off attached ElasticBlockStore volumes does not exceed a maximum value (by default, 39, since Amazon reccomends a maximum of 40).
```

Comment:
At least GCE hs similar limitation of how many volumes they can attach to a node (I'm not sure about OpenShift). I think it would be better to have single generic command line option for all of them instead of individual  `MaxEBSVolumeCount`/`MaxPDVolumeCount` (`MaxCinderVolumeCount`).